cdap-master/src/main/java/co/cask/cdap/data/runtime/main/MasterServiceMain.java

```diff
@@ -127,7 +127,7 @@
   private final Configuration hConf;
   private final Injector baseInjector;
   private final ZKClientService zkClient;
-  private final TwillRunnerService twillRunner;
```

Comment:
I moved the declaration to the `LeaderElection` class. However this requires passing `TwillRunnnerService` instance instance around. Can you please check again?